src/components/FormContainer.jsx

```diff
@@ -57,20 +59,17 @@ const FormContainer = (props) => {
   };
 
   return (
-    <div className="bc-citysourced-reporter">
-      <ol className="bc-citysourced-reporter-steps">
-        {tabList.map((tab, id) => {
+    <div>
+      <ol className="dg_progress-tabs">
```

Comment:
I'm assuming we didn't use the component here, because we didn't export the item component?

Reply:
per our chat, the components tabs are working the way we want, but tabs within this project don't adhere to that style. Therefore we just need to take advantage of the css until we update this project.